libs/blocks/notification/notification.css

```diff
@@ -340,6 +340,16 @@
   display: none;
 }
 
+.visibility-hidden {
```

Comment:
The class name is general enough that might affect other elements if the code gets loaded. Can we make it more specific, like `notification-visibility-hidden` or `notification-hidden-wrapper`